dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/subsystem/FeatureDefinitionUtils.java

```diff
@@ -85,7 +85,8 @@
     public final static Collection<String> ALLOWED_ON_CLIENT_ONLY_FEATURES = Arrays.asList("com.ibm.websphere.appserver.javaeeClient-7.0",
                                                                                            "com.ibm.websphere.appserver.javaeeClient-8.0",
                                                                                            "io.openliberty.jakartaeeClient-9.0",
-                                                                                           "com.ibm.websphere.appserver.appSecurityClient-1.0");
+                                                                                           "com.ibm.websphere.appserver.appSecurityClient-1.0",
+                                                                                           "io.openliberty.appSecurityClient-2.0");
```

Comment:
There is no appSecurityClient-2.0 feature

Reply:
Changed.

I had originally created one before the private servlet feature rename. Had missed this change from prior to merging in those changes.